app/views/admin/claims/show.html.erb

```diff
@@ -10,6 +10,12 @@
       </div>
     <% end %>
 
+    <% if @claim_ids_with_matching_attributes.present? %>
```

Comment:
@tekin picked up something similar on a PR of mine: given that we know that `@claim_ids_with_matching_attributes` is a collection (something hash-like), I think it would be clearer if we had

```suggestion
    <% if @claim_ids_with_matching_attributes.any? %>
```

(same a few lines down, too)